manifests/params.pp

```diff
@@ -6,7 +6,7 @@
     'Archlinux' : {
       $cmdacl            = ['cmdallow 127.0.0.1']
       $config            = '/etc/chrony.conf'
-      $config_template   = 'chrony/chrony.conf.archlinux.erb'
+      $config_template   = 'chrony/chrony.conf.epp'
```

Comment:
Since this is now the same everywhere, you can move it to `init.pp` as a static value.

Reply:
Yes absolutely, but I think it should have it's own commit, and a separate PR

Reply:
I disagree. init.pp already has all static values and params.pp only has values that differ for various OSes.

Reply:
Ok, I see your point I have added a commit to do this.